datasets/pl/mswia_sanctions/crawler.py

```diff
@@ -74,11 +73,11 @@ def crawl_row(context: Context, row: Dict[str, str], table_title: str):
 
     entity.id = context.make_slug(table_title, name)
     names = name.split("(")
-    entity.add("name", names[0])
+    entity.add("name", names[0], lang="pol")
     for alias in names[1:]:
         entity.add("alias", alias.split(")")[0])
     notes = row.pop("uzasadnienie_wpisu_na_liste")
-    entity.add("notes", notes)
+    entity.add("notes", notes, lang="pol")
```

Comment:
Should we just mark the whole dataset as polish in `data.lang` of the YAML?

Reply:
sure, it was already marked in the `data.lang`, I missed it